ztp/ztp-ran-policy-generator/kustomize/plugin/ranPolicyGenerator/v1/ranpolicygenerator/policyGen/policyHelper.go

```diff
@@ -76,17 +77,27 @@ func CreatePlacementRule(name string, namespace string, matchKey string, matchOp
 	placmentRule.Kind = "PlacementRule"
 	placmentRule.Metadata.Name = name + "-placementrule"
 	placmentRule.Metadata.Namespace = namespace
-	expressions := make(map[string]interface{})
+	expression := make(map[string]interface{})
 	if matchOper == utils.ExistOper {
-		expressions["key"] = matchKey
-		expressions["operator"] = matchOper
+		expression["key"] = matchKey
+		expression["operator"] = matchOper
 	} else {
-		expressions["key"] = matchKey
-		expressions["operator"] = matchOper
+		expression["key"] = matchKey
+		expression["operator"] = matchOper
```

Comment:
if this is done on both side of if, it can be done before the if. 
so 
```
expression["key"] = matchKey
expression["operator"] = matchOper
if matchOper != utils.ExistOper {
    expression["values"] = strings.Split(matchValue, ",")
}
```
is cleaner and shorter